setup.py

```diff
@@ -41,7 +41,13 @@ def main():
         packages=setuptools.find_packages(),
         python_requires=">=3.6",
         zip_safe=False,
-        install_requires=["iso639", "requests", "requests-html", "setuptools"],
+        install_requires=[
+            "iso639",
+            "segments",
```

Comment:
It looks like they follow semantic versioning (see [their changelog](https://github.com/cldf/segments/blob/393aef48423ed56660052ea583d2e13bd3fcd3e4/CHANGES.md)), and development is still pretty active (last commit on master was from yesterday). Perhaps we should have `"segments < 3"` here, just in case they have API-breaking changes?

(I think the other packages in `install_requires` are pretty stable and therefore it's fine to not have an upper bound for version numbers.)

Reply:
Didn't realize `install_requires` supported that. Done.